kDrive/UI/Controller/Files/File List/FileListViewModel.swift

```diff
@@ -93,6 +93,7 @@ class FileListViewModel: SelectDelegate {
         var matomoViewPath = ["FileList"]
     }
 
+    var dismissClosure: (() -> Void)?
```

Comment:
There is already a naming pattern and a "region" to declare callbacks, see:
https://github.com/Infomaniak/ios-kDrive/blob/c35026c147d0c9811d2cde3990fbd55ab13a156d/kDrive/UI/Controller/Files/File%20List/FileListViewModel.swift#L136